crates/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -266,6 +337,22 @@ tensor_type!(arrow2::types::f16, F16);
 tensor_type!(f32, F32);
 tensor_type!(f64, F64);
 
+// Manual expension of tensor_type! macro for `u8` types. We need to do this, because u8 can store encoded data
+impl<'a> TryFrom<&'a TensorData> for ::ndarray::ArrayViewD<'a, u8> {
```

Comment:
this covers how to go from TensorData -> ArrayView, but the other way around is not defined now, causing a bunch of tests to fail compiling

Reply:
I'm splitting the macro up to fix it :)

Reply:
Ah, thanks!